Sources/XiEditor/RPCSending.swift

```diff
@@ -202,13 +206,9 @@ class StdoutRPCSender: RPCSending {
 
     private func sendJson(_ json: Any) {
         do {
-            let data = try JSONSerialization.data(withJSONObject: json, options: [])
-            let mutdata = NSMutableData()
-            mutdata.append(data)
-            let nl = [0x0a as UInt8]
-            mutdata.append(nl, length: 1)
-            rpcLogWriter?.write(bytes: mutdata as Data)
-            inHandle.write(mutdata as Data)
+            let data = try JSONSerialization.data(withJSONObject: json, options: []) + NEW_LINE
+            self.rpcLogWriter?.write(bytes: SEND_LOG_PREFIX + data)
```

Comment:
Do we know that `+` on two `Data` doesn't reallocate?

Reply:
Good call @cmyr.  I hadn't thought of that!  I've pushed an update....

Reply:
FYI, I did some profiling and it looks like the `+` does re-alloc. Does this look better?

Reply:
Yep!